### DISCUSSION TITLE
Compatible with Cloudflare Workers

### DISCUSSION THREAD

**Discussion by elmarw** [harttle/liquidjs#460] [Q&A]
Would it be possible to make liquid.js work for Cloudflare Workers? It looks like a perfect match because Cloudflare Workers does not allow the use of eval and new Function. I did some tests and everything works quite nicely except for including other .liquid files which requires a Node fs path library.

Possible solution might be an option to include the templates in the function?

```
import partial from "./views/partial.liquid"
var engine = new Liquid({
    partials: {
      'partial.liquid' : partial
    }
});
```

And call for example in a liquid file:
```
{% render "partial" %}
```

**Comment by elmarw** [comment]
I just found out that you can replace the FS. Great addition and solves my issue. Thanks!

**Comment by harttle** [comment]
I'd like to add a link for the Abstract FileSystem doc (it's not complete but points out how to do it): https://liquidjs.com/tutorials/render-file.html
